app/models/ability.rb

```diff
@@ -48,14 +48,15 @@ def initialize(user)
       can [:create], AgreementLetter
       can [:new, :create], Request
       cannot :view_personal_details, ApplicationLetter, user: { id: !user.id }
+      can [:show, :index, :archive], Event
```

Comment:
I understand, but I think it's cleaner not to skip it. As a developer, I expect to be able to use the abilities to control the behavior of all of the actions. Skipping authorization just because it's not needed right now feels like introducing technical debt to me.